src/main/java/TodoCommand.java

```diff
@@ -35,15 +35,26 @@ String execute(TaskList tasks, Storage storage) throws DukeException, IOExceptio
                 Todo newTodo = new Todo(commandDesc.substring(5));
                 storage.appendToFile(System.getProperty("user.dir") + "/data/tasks.txt", newTodo.stringForAppend());
                 tasks.addToDo(newTodo);
-                String output = "Got it. I've added this task: \n";
-                output += (newTodo + "\n");
-                output += "Now you have " + tasks.taskList.size() + " tasks in the list.\n";
-                return output;
+                return getOutput(tasks, newTodo);
             } else {
                 throw new EmptyDescDukeException("todo");
             }
         } catch (IndexOutOfBoundsException err) {
             throw new EmptyDescDukeException("todo");
         }
     }
+
+    /**
+     * Forms the output string.
+     *
+     * @param tasks TaskList containing the tasks
+     * @param newTodo new Todo to be added
+     * @return output string
+     */
+    private String getOutput(TaskList tasks, Todo newTodo) {
+        String output = "Got it. I've added this task: \n";
+        output += (newTodo + "\n");
+        output += "Now you have " + tasks.taskList.size() + " tasks in the list.\n";
```

Comment:
Good job on your code! Your variables are very appropriately named. The only thing I can suggest for improvement is maybe tailoring the output line to print "task" or "tasks" depending on whether the number of tasks in your list is <= 1 or otherwise